src/main.ts

```diff
@@ -0,0 +1,24 @@
+import { getInput, setFailed } from '@actions/core';
+import { Octokit } from '@octokit/core';
+
+import '@total-typescript/ts-reset';
+
+import action from './action';
+
+const octokit = new Octokit({
+  auth: getInput('github_token', { required: true }),
+});
+
+try {
+  await action(octokit);
```

Comment:
Can you please add there a comment, where is action called? Even it is file action.ts.

Reply:
Sure, done